feeds.conf

```diff
@@ -1,7 +1,7 @@
 src-git packages https://github.com/openwrt/packages.git^327a03e8f0c61f8ee98adbee5bd37ce7bccbb049
 src-git luci https://github.com/openwrt/luci.git^5ec72617eede467c932642b60291d627fca70342
 src-git routing https://github.com/openwrt-routing/packages.git^1b9d1c419f0ecefda51922a7845ab2183d6acd76
-src-git packages_berlin https://github.com/freifunk-berlin/firmware-packages.git^b0c64387a8bb5894b7efeaeff75b389009b656bb
+src-git packages_berlin https://github.com/freifunk-berlin/firmware-packages.git^317b65600e830be1d9f0e751cb5bbda137aa2540
 src-git gluon https://github.com/freifunk-gluon/packages.git^c34d129afb48f225741b683127beee6ba3c169d2
```

Comment:
normally the firmware master-branch only references to the master-branch of the packages-repo

Reply:
Yes, and when the openvpn245 branch in the packages-repo is merged, then this merge should end up pointing to the master branch, right?

Shall I drop the commit to feeds.conf?